src/aat/java/uk/gov/hmcts/ccd/datastore/befta/DataStoreBeftaMain.java

```diff
@@ -0,0 +1,12 @@
+package uk.gov.hmcts.ccd.datastore.befta;
+
+import uk.gov.hmcts.befta.BeftaMain;
```

Comment:
it would be good to add some quick javadoc at the class level that explain the purpose of various main classes of the framework

Reply:
Sure.. Ticket created for this just now. (RDM-7470)